dockyard/api/app.py

```diff
@@ -0,0 +1,42 @@
+import pecan
+
+from dockyard.api import config as api_config
+from oslo_config import cfg
+from pecan import make_app
+
+from dockyard import model
+
+
+API_SERVICE_OPT = [
+    cfg.PortOpt('port',
+                 default=5869,
+                 help='Port for the dockyard service.'),
+    cfg.IPOpt('host',
+              default='127.0.0.1',
+              help='Listening address for dockyard service'),
+]
+
+CONF = cfg.CONF
+opt_group = cfg.OptGroup(name='default',
+                         title='Group for the default values for dockyard api')
+CONF.register_group(opt_group)
+CONF.register_opts(API_SERVICE_OPT, opt_group)
+
+def get_pecan_config():
```

Comment:
What is the purpose of this function, we have to identify and then remove if not required.